snapshot/git/gitdb/bundlestore.go

```diff
@@ -330,14 +330,18 @@ func (b *bundlestoreBackend) uploadFile(filePath string, ttl *store.TTLValue) (s
 	if err != nil {
 		return "", err
 	}
+	fi, err := f.Stat()
+	if err != nil {
+		return "", err
+	}
 	defer f.Close()
```

Comment:
we still want to Close() even if f.Stat() fails, right?

Reply:
yes, thanks